### PR TITLE
[#795] Fix reset page after changing number of items on page

### PR DESCRIPTION
**Pull request by goreck888**
Now after change number of items on page, page is reset to 1.

### CODE REVIEW

**Comment by mkasztelnik** [comment]
@goreck888 [this](https://github.com/cyfronet-fid/marketplace/blob/master/app/views/services/_paginate_simple.haml) is a place you need to fix instead of modifying a controller. You need to take all query params instead of `page`, something like:

```ruby
params.merge(per_page: 20).permit!.without(:page)
```

**Comment by goreck888** [comment]
> @goreck888 [this](https://github.com/cyfronet-fid/marketplace/blob/master/app/views/services/_paginate_simple.haml) is a place you need to fix instead of modifying a controller. You need to take all query params instead of `page`, something like:
> 
> ```ruby
> params.merge(per_page: 20).permit!.without(:page)
> ```

Yeah, the first solution was change it in haml but there was a problem with tests on brakeman, so that's the reason I changed it in controller.

**Comment by mkasztelnik** [comment]
Than please fix brakeman issue :smile:

**Comment by goreck888** [comment]
> pr-801 get me to marketplace-12
> i try on allservices page get to second, third or 6th page with no success. please, fix this :)

I fixed that :)

**Comment by goreck888** [comment]
> Then please fix brakeman issue smile

i don't know if it's good solution for the brakeman, but I saw it's similar like with per_page issue.

**Comment by mkasztelnik** [comment]
> > Then please fix brakeman issue smile
> 
> i don't know if it's good solution for the brakeman, but I saw it's similar like with per_page issue.

This is false positive case and should be added to brakeman ignore list. Use [`brakeman -I`](https://brakemanscanner.org/docs/ignoring_false_positives/) to add these cases to ignorelist.

**Comment by goreck888** [comment]
> > > Then please fix brakeman issue smile
> > 
> > 
> > i don't know if it's good solution for the brakeman, but I saw it's similar like with per_page issue.
> 
> This is false positive case and should be added to brakeman ignore list. Use [`brakeman -I`](https://brakemanscanner.org/docs/ignoring_false_positives/) to add these cases to ignorelist.

That's what I did :)

File: config/brakeman.ignore [hunk 1]
```diff
@@ -10,7 +10,7 @@
       "line": 15,
       "link": "https://brakemanscanner.org/docs/warning_types/dynamic_render_path/",
       "code": "render(action => policy_scope(Project).order(:name).eager_load(:project_items).where(\"project_items.status = ?\", params[:status]), {})",
-      "render_path": [{"type":"controller","class":"ProjectsController","method":"index","line":9,"file":"app/controllers/projects_controller.rb","rendered":{"name":"projects/index","file":"/Users/andrzej/RubymineProjects/marketplace/app/views/projects/index.html.haml"}}],
+      "render_path": [{"type":"controller","class":"ProjectsController","method":"index","line":9,"file":"app/controllers/projects_controller.rb","rendered":{"name":"projects/index","file":"/home/goreck/workspace/RubymineProjects/marketplace/app/views/projects/index.html.haml"}}],
```
[the remark this answers is not in the document]
Comment (goreck888) [reply config/brakeman.ignore]: I just used manager inside brakeman and I don't know, how it works but after add an exception in that way, test passed in docker.

**Comment by goreck888** [comment]
> please rebase because there are only 5 services right now.

Done.

**Pull request merged**